Apps/ClinicalDocument/src/Services/ClinicalDocumentService.cs

```diff
@@ -121,10 +121,9 @@ public async Task<RequestResult<EncodedMedia>> GetFileAsync(string hdid, string
                 RequestResult<PersonalAccount> response = await this.personalAccountsService.GetPatientAccountResultAsync(hdid).ConfigureAwait(true);
                 if (response.ResultStatus == ResultType.Success)
                 {
-                    string? pid = response.ResourcePayload?.PatientIdentity.Pid.ToString();
+                    Guid? pid = response.ResourcePayload?.PatientIdentity.Pid ?? throw new InvalidOperationException($"Pid not found for hdid {hdid}");
```

Comment:
I think it's possible to make the variable non-nullable now.

Reply:
Fixed, thanks. +10 for var.